.cookiecutter.json

```diff
@@ -20,6 +20,7 @@
     "open_source_license": "Apache-2.0",
     "docs_base_url": "https://docs.nautobot.com",
     "docs_app_url": "https://docs.nautobot.com/projects/floor-plan/en/latest",
-    "_template": "./cookiecutter-ntc/nautobot-plugin"
+    "_template": "https://github.com/networktocode-llc/cookiecutter-ntc.git",
+    "_output_dir": "/opt/ntc/networktocode-llc/cookiecutter-ntc/tools/ntc-cookie-drift-manager/outputs"
```

Comment:
Should probably delete this. A bunch of references to a private repo.

Reply:
`.cookiecutter.json` file will be replaced with `cookie.json` file, that doesn't contain this local info. Removed manually now. I'll update drift manager to be less verbose to PR description as well.

Reply:
Moved back to `.cookiecutter.json` after consulted with @cmsirbu